PHP.md

```diff
@@ -304,4 +304,215 @@ $closureWithArgs = function ($arg1, $arg2) {
 $closureWithArgsAndVars = function ($arg1, $arg2) use ($var1, $var2) {
     // body
 };
-```
\ No newline at end of file
+```
+
+### Convention of method names
+
+#### The methods name should start with the verb
+```php
+//bad
+public function firstName()
+{
+
+}
+
+
+//good
+public function getFirstName()
+{
+
+}
+
+```
+#### Basic verbs to be used in the first order and when creating a method name

```

Comment:
I think that we need more explanation in this paragraph. I propose to write that each method which does some activity  should start with a prefix as a verb